simple_regression/simple_regression.Rmd

```diff
@@ -186,10 +187,18 @@ is linear using a scatterplot. We should also verify this condition with a plot
 of the residuals vs. at-bats.
 
 ```{r residuals}
-qplot(x = .fitted, y = .stdresid, data = m1)
+qplot(x = .fitted, y = .resid, data = m1) +
+  geom_hline(yintercept = 0, linetype = "dashed") +
```

Comment:
Once we have the drafts of all of the labs done, we'll need to go back through and see if/when/how we want to introduce the '+'.